src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -894,6 +943,17 @@ def _revXlateList(self, model, yang, config, table):
                 # fill rest of the entries
                 for key in entry:
                     if key not in pkeydict:
+                        if ccontainer and key == ccontainer['@name']:
```

Comment:
>'@name' [](http://example.com/codeflow?start=60&length=7)

Will it throw is '@name' is not in ccontainer?

Reply:
indeed